sql/CMakeLists.txt

```diff
@@ -144,7 +144,7 @@ add_custom_target(sqlfile ALL DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/${INSTALL_FILE
 set(MOD_FILE_REGEX "([0-9]+\\.[0-9]+\\.*[0-9]+[-a-z0-9]*)--([0-9]+\\.[0-9]+\\.*[0-9]+[-a-z0-9]*).sql")
 
 # We'd like to process the updates in reverse (descending) order
-set(MOD_FILES_LIST "${MOD_FILES_VERSIONED};${UPDATE_FROM_VERSION}--${PROJECT_VERSION_MOD}.sql")
+set(MOD_FILES_LIST ${MOD_FILES_VERSIONED})
```

Comment:
Any point in renaming this var? Can't we just use `MOD_FILES_VERSIONED` everywhere `MOD_FILES_LIST` would be used then?

Reply:
Agreed

Reply:
@RobAtticus dunno, but's it's been good enough for the past year, and I don't think it's a priority right before. Remind me after, and I'll look into refactoring the file then.

Reply:
With the change now it's less necessary / may not even work.